pytest/common_lib/shared/__init__.py

```diff
@@ -268,14 +276,24 @@ def generate_mpc_configs(
         "--desired-responder-keys-per-participant",
         str(num_respond_aks),
     )
+
+    if len(migrating_nodes) > 0:
+        cmd = cmd + (
+            "--migrating-nodes",
+            ",".join(f"{i}" for i in migrating_nodes),
+        )
+
     if presignatures_to_buffer:
         cmd = cmd + (
             "--desired-presignatures-to-buffer",
             str(presignatures_to_buffer),
         )
     subprocess.run(cmd)
 
-    candidates = []
+    for i in migrating_nodes:
+        responders.append(responders[i])
```

Comment:
is this correct? looks quite suspicious 😶‍🌫️

Reply:
This is very much intended. Let me see if I can circle back to this comment later for an explanation.